helm_remote/Tiltfile

```diff
@@ -91,14 +93,11 @@ def helm_remote(chart, repo_url='', repo_name='', release_name='', values=[], se
     else:
         k8s_yaml(helm(chart_target, name=release_name, values=values, set=set))
 
-
 def install_crds(name, directory):
     name += '-crds'
+    files = str(local(r"grep --include '*.yaml' --include '*.yml' -rEil '\bkind\s*:\s+CustomResourceDefinition\s*$' %s || exit 0" % directory)).strip()
 
-    is_mac = str(local('uname -s')).strip().lower() == 'darwin'
-
-    files = str(local(r"grep --include '*.yaml' --include '*.yml' -rEil '\bkind\s*:\s+CustomResourceDefinition\s*$' %s" % directory + " | xargs" + ("" if is_mac else " -r") + " grep -L '^{{-'")).strip()
```

Comment:
The `grep -L '^{{-'` from this line was removed from the new line (98) above -- this is critical for filtering out any helm _template_ files that may contain the string `CustomResourceDefinition` but are simply templates rather than actual CRDs

Reply:
If the goal is to obviate the call to `xargs`, this should work instead:
```
files = str(local(r"grep -L '^{{-' $(grep --include '*.yaml' --include '*.yml' -rEil '\bkind\s*:\s+CustomResourceDefinition\s*$' %s) || exit 0" % directory)).strip()
```